api/v1beta2/rediscluster_types.go

```diff
@@ -27,6 +27,9 @@ import (
 type RedisClusterSpec struct {
 	Size             *int32           `json:"clusterSize"`
 	KubernetesConfig KubernetesConfig `json:"kubernetesConfig"`
+	HostNetwork      bool             `json:"hostNetwork,omitempty"`
+	// +kubebuilder:default:=6379
+	Port int32 `json:"port,omitempty"`
```

Comment:
```suggestion
	Port  *int `json:"port,omiteimpty"`
```

Reply:
Make this pointer we can check if the port==nil then we se the default instead of checking it of 
port==0